README.md

```diff
@@ -123,6 +123,10 @@ data "http" "production_cp" {
 }
 ```
 
+## Terraform Compatibility and Versioning
```

Comment:
Does it make sense to have this on our [developer site](https://developer.konghq.com/terraform/)?